server/server.js

```diff
@@ -36,6 +36,38 @@ var announce_timer = 0;
 /* used for triggering sound effects */
 var oldLightCapturePointCount = 0;
 var oldDarkCapturePointCount = 0;
+var commands = {
+  "!bspeed" : function(params){
+    var value = parseFloat(params);
+    if(value){
```

Comment:
Doing validity checking like this will also stop 0 as an "invalid" value. Intended? Consider using isNaN to check the output of parseFloat instead.